linters/nelua.lua

```diff
@@ -0,0 +1,36 @@
+-- Nelua plugin for lint+
+
+--- CONFIG ---
+
+-- config.lint.nelua_mode: "analyze" | "lint"
+--   changes the linting mode, "analyze" (default) does a complete checking,
+--   while "lint" only checks for syntax errors.
+
+--- IMPLEMENTATION ---
+
+local core = require 'core'
+local lintplus = require 'plugins.lintplus'
+
+local mode = lintplus.config.nelua_mode or "analyze"
+
+if mode ~= "analyze" and mode ~= "lint" then
+  core.error("lint+/nelua: invalid nelua_mode '%s'. Available modes: 'analyze', 'lint'", mode)
```

Comment:
If we only accept these two flags as correct, we probably want to set a fallback for `mode` to prevent invalid argments being passed to `nelua`.

Reply:
Indeed, done at 9424ea8